### PR TITLE
Add new logo, and fix issue with Goodbye message

### PR DESCRIPTION
**Pull request by reginafcompton**
This pull request:
- adds a simplified logo in the navbar
- integrates a secondary font
- fixes an issue, so that the user gets the correct "Goodbye" message

### CODE REVIEW

File: templates/ces_client/index.html [hunk 1]
```diff
@@ -12,9 +12,14 @@
...
-        <p>Are you looking for a place to stay tonight? Are you experiencing some form of homelessness? Talk to B.E.N. He will help guide you to your next location. Click "Goodbye!" or refresh the page.</p>
+        <div class='row'>
+            <div class='col'>
+                <a class="img-fluid float-left mr-3" href="{% url 'index' %}"><img src="{% static 'images/CESlogo.svg' %}" width='60'></a>
+                <h3>Connect to End Homelessness</h3>
+                <h4>A unified community response</h4>
+            </div>
+        </div>
+        <p>This site helps people experiencing homelessness find appropriate resources. If you need help, please talk to the bot B.E.N. in the chat window below. He will ask you questions and, when possible, recommend a phone number to call or address to go to. If you want to exit, then click "Goodbye".</p>
     </div>
```
Comment (jmithani) [inline templates/ces_client/index.html]: I would suggest refactoring this as two columns. I would also make the logo larger. I hacked with some inline styles to give an idea of what I mean:

screen shot 2019-01-08 at 2 44 15 pm
screen shot 2019-01-08 at 2 44 04 pm

With perhaps more margin above the paragraph text.

File: templates/ces_client/index.html [hunk 2]
```diff
@@ -12,9 +12,14 @@
 
 <div class='row'>
```
Comment (jmithani) [inline templates/ces_client/index.html]: I think less white space between the content and the navbar could work.

**Comment by jmithani** [comment]
oh! i would also change the favicon to the door logo and change the head text (right now it's just 'Index')

File: ces_client/utils.py [hunk 3]
```diff
@@ -82,7 +82,7 @@ def prep_text_for_botui(text):
         The web interface also requires users to "click" rather than "type."
         This function changes the text accordingly.
         '''
-        text_with_br = text.replace('\n', '<br>')
+        text_with_br = text.replace('\r\n\r\n', '\n').replace('\n', '<br><br>')
         prepped_text = text_with_br.replace('Type', 'Click')
 
         return prepped_text
```
[the remark this answers is not in the document]
Comment (reginafcompton) [reply ces_client/utils.py]: I'll open an issue for this! Good eye.

**Comment by reginafcompton** [comment]
@jmithani - I believe I've addressed your feedback (favicon included, too!). Let me know how this looks.

File: templates/base.html [hunk 4]
```diff
@@ -8,35 +8,32 @@
...
+    <link rel="stylesheet" href="{% static "css/custom.css" %}" type="text/css" />
 </head>
 <body>
 
     <nav class="navbar navbar-expand-lg">
-        <a class='navbar-brand' href="{% url 'index' %}">
-            <img src="{% static 'images/connect-icon.png' %}" class="img-fluid" width="110px">
-            <!-- Connect to End Homelessness -->
-        </a>
         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (jmithani) [inline templates/base.html]: There is some funky coloring on the responsive view of the navbar. I think the hamburger icon is white, which is why it isn't appearing. (Highlighting is my own).
screen shot 2019-01-09 at 10 28 54 am
screen shot 2019-01-09 at 10 29 06 am

File: coordinated-entry-screening/static/css/custom.css [hunk 5]
```diff
@@ -83,8 +76,21 @@ nav {
...
+    font-size: 16px;
+}
+
+.bot-msg div {
+    padding-bottom: 4px;
+}
+
+.botui-message-content {
+    line-height: 1.4;
+    max-width: 70%;
```
Comment (jmithani) [inline coordinated-entry-screening/static/css/custom.css]: We should add a media query here for small screens; the text becomes unreadable.
screen shot 2019-01-09 at 10 29 27 am
screen shot 2019-01-09 at 10 29 39 am

Comment (reginafcompton) [reply coordinated-entry-screening/static/css/custom.css]: I am not sure the best solution. I increased the size of the content box (for smaller devices). Does that make sense?

Comment (jmithani) [reply coordinated-entry-screening/static/css/custom.css]: Yes! I would also scale B.E.N.'s avatar, lower the font size, and possibly lower the line height. The latter two might be easier if we define them via rem values (which might eliminate the need for a media query, I'm not totally sure).

Articles:
[rem values in-depth](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/)
[much more concise breakdown](https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97)

**Pull request merged**